internal/http/client.go

```diff
@@ -72,9 +79,8 @@ func (c *Client) Do(method string, path string, requestBody, responseBody interf
 	if err == nil {
 		req.Header.Set("x-request-id", id.String())
 	}
-	req.Header.Set("Authorization", "Bearer "+c.Metadata.AuthToken)
 	req.Header.Set("X-Cli-Version", c.Metadata.CLIVersion)
-	req.Header.Set("X-Caller-Email", c.Metadata.CallerEmail)
+	req.Header.Set("X-HAM-TOKEN", "test")
```

Comment:
Is this intentional?

Reply:
Yes, Im an still in draft, so i figures i would just make tests pass and then get some feedback regarding the ude of this token (as mentioned in the PR) before we do something about it ✌️ this Will most definetely make it to the final change set 😅